.buildkite/pipeline.yml

```diff
@@ -11,3 +11,8 @@ steps:
 
   - label: Build the pex file
     command: mkdir -p dist && .buildkite/build_pex.sh
+
+  - wait
```

Comment:
Oh, and no need for this wait clause. By removing it we can build concurrently with the pex file.